docs/source/guide/zne-1-intro.myst

```diff
@@ -0,0 +1,144 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.3
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# How do I use ZNE?
+
+Zero noise extrapolation is an error mitigation technique in which an expectation 
```

Comment:
Hyphenate zero-noise for consistency.

Reply:
Also, it's obvious, but good practice to put "Zero-noise extrapolation (ZNE)" to formally define the acronym.

Reply:
Updated, thanks!